src/applications/mhv/secure-messaging/containers/AuthorizedRoutes.jsx

```diff
@@ -13,7 +13,11 @@ import { Paths } from '../util/constants';
 
 const AuthorizedRoutes = () => {
   return (
-    <div className="vads-u-flex--fill" data-testid="secure-messaing">
+    <div
+      className="vads-l-col--12
+      medium-screen:vads-l-col--9"
+      data-testid="secure-messaing"

```

Comment:
messaging is missed spelled 👀

Reply:
it looks like it was there before you made changes but we should correct it and update it's associated tests as well.